main.go

```diff
@@ -28,13 +29,14 @@ func main() {
 	// Required flags are: -file string, -line int.
 	// Optional flag is; -cache_root string
 	// No other argument is needed.
+	server := flag.String("server", defaultServer, "Server name [REQUIRED] ")
```

Comment:
Should `-server` be a required or optional argument? I don't think it needs to be required.